Add Governance page (#118)
LiskHQ/lisk-docs: docs/about-lisk/governance.mdx

---
title: Governance
sidebar_position: 1
slug: /governance
description: Documentation about the governance of the Lisk project.
keywords:
  [
    Lisk governance,
    Lisk DAO,
    Lisk Grants,
    Funding,
    Delegation,
    Voting,
    Proposals,
  ]
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

# Governance

> At the moment, the security of the Lisk Mainnet is dependent on a [multisig wallet](https://www.coindesk.com/tech/2020/11/10/multisignature-wallets-can-keep-your-coins-safer-if-you-use-them-right/) managed by several individuals of the Onchain Foundation.

Reviewer: > individuals of the Onchain Foundation

Jan and Shu are also members of the group but do not belong to the Foundation.

Author: Should we mention the Security council instead? If yes, do you have a good explanation how to explain it to the public?

Author: @AndreasKendziorra I updated the section according to Jans feedback!

Reviewer: should we rather use "key members" instead of "individuals" as Jan proposed? The current way sounds a bit like arbitrary community members are involved, and then readers might wonder how they could get a chance to become a member.

To decentralize the governance of the Lisk project further, Lisk introduces a decentralized autonomous organization, the Lisk DAO, which allows Lisk token holders to participate together in the governance of the Lisk project.

:::info
Every LSK token holder who participates in the DAO is rewared with [staking rewards](#staking-rewards), to incentivize participation in the Lisk governance.
:::

## Lisk DAO
The Lisk DAO provides the opportunity for LSK token holders to participate in the overall governance of the Lisk project, including allocation of funds, protocol updates, or strategic directions. Decisions are made via [proposals](#proposals).

### Tools
The following tools are available to participate in the Lisk DAO:

<Tabs>
  <TabItem value="tally" label="Tally">
  Tally is the platform used to create and vote for proposals of the Lisk DAO.

  {/* TODO: Update link to tally instance */}
  The [Lisk DAO Tally instance](https://www.tally.xyz/gov/3rd-testing) is the place where users can vote on proposals or delegate their voting power.
  </TabItem>
  <TabItem value="forum" label="Forum">

>   The [Lisk Governance Forum](https://forum.lisk.com/) is the place for the community to discuss the Lisk gonvernance and draft proposals, before they are submitted on Tally.

Reviewer: ```suggestion
  The [Lisk Governance Forum](https://forum.lisk.com/) is the place for the community to discuss the Lisk governance and draft proposals, before they are submitted on Tally.
```

  </TabItem>
  <TabItem value="discord" label="Discord">

>   The `#lisk-dao` channel in the [Lisk Chat](https://lisk.chat) Discord server is a comunity chat dedicated to discuss topics all around the Lisk DAO, share news and updates, or share ideas for new proposals.

Reviewer: ```suggestion
  The `#lisk-dao` channel in the [Lisk Chat](https://lisk.chat) Discord server is a community chat dedicated to discuss topics all around the Lisk DAO, share news and updates, or share ideas for new proposals.
```

  </TabItem>
</Tabs>

### Treasury
{/* TODO: Update link to timelock contract */}

> The Lisk DAO has an associated treasury, which resides in the [TimeLock contract](https://sepolia-blockscout.lisk.com/address/0x4b727D3f74DD4c124Ced030335DebD5e7EBB8366).

Reviewer: ```suggestion
The Lisk DAO has an associated treasury, which resides in the [TimelockController contract](https://sepolia-blockscout.lisk.com/address/0x4b727D3f74DD4c124Ced030335DebD5e7EBB8366).
```

The current balance of the treasury can be checked on the [Lisk BlockScout Explorer](https://sepolia-blockscout.lisk.com/address/0x4b727D3f74DD4c124Ced030335DebD5e7EBB8366).
The treasury is NOT managed by the Onchain Foundation.
Instead, the funds from the treasury can only be transferred via successful [funding proposals](#proposal-types).

Tokens for the Lisk DAO fund are minted newly and added to the Lisk DAO according to the following vesting plan:

| Year | Amount (LSK) | Description |
|------|--------------|-------------|
| 2024 | 15,000,000 | 6,250,000 LSK liquid at migration, 8,750,000 LSK are linearly released in 2024, starting from the migration. |
| 2025 | 15,000,000 | LSK are linearly released over the year. |
| 2026 | 15,000,000 | LSK are linearly released over the year. |

:::note
The DAO fund may be increased further in the future by additional 100,000,000 LSK over the following 7 years, if the DAO members approve the corresponding proposals.
:::

> ### Staking rewards

Reviewer: capitalization of headers is not consistent in this document. E.g., we have "Proposal Evaluation" below.

Reviewer: For headers of level 2 (`Governance Goals` and `Design Principles`), you still use capitalization. Is this intended?

Author: Yes, imho it looks nicer. What do you think?

Reviewer: I think it's common to use capitalization for headers. Not only for certain levels, but for all. So, I would apply it to all headers. But I don't know how you handle it in general in the repository. Consistency over the repository is more important I would say.

Author: OK, I adjusted it now to lower case, as this is used currently in most cases.

Staking rewards incentivize users to participate in governance, making the decision making more decentralized and community-driven.  

> In short, users get rewards for locking/staking tokens for governance participation.

Reviewer: this might be a bit confusing for readers as it wasn't explained yet that one actually has to stake to participate. I would propose to move the _staking rewards_ section to after _Delegation_ or _Administrative role of the Onchain Foundation_.

We apply a **time-weight boosting** on rewards:
The longer locking period a user sets, the more rewards they get.
This way, we reward token holders for committing long-term to the Lisk project so we attract them as long-term advocates for Lisk while reducing short-term speculation and volatility. 

> - Users can modify their locking positions (increase amount, pause locking duration) at any time they want.

Reviewer: they can also extend the duration

- Users can claim their rewards at any time.   
- Users have the option to **pause** their locking period countdown.
That means, the remaining locking duration remains fixed until the user decides to resume it.

> Note that this results in **higher staking rewards**, since the weight remains fixed, as opposed to a weight daily decreasing when the countdown is active. 

Reviewer: > opposed to a weight daily decreasing

it was not clearly explained before that the weight is decreasing. This could be done by changing line 77 to:

> The further the end of the locking duration is in the future, the more rewards they get. This also implies that the rewards decrease every day.

#### Staking rewards distribution

> {/* TODO: Add Blockscout link to address of the Staking contract */}

Reviewer: the address of the Reward contract might be more meaningful in this case

Author: what is the reward contract? it dont see it in the DAO contract specs

Reviewer: this reminds me that I must update the confluence page. Sorry for this, I will do it very soon.
The staking functionality was eventually split into 3 different contracts: `LockingPositionContract`, `StakingContract` and `RewardContract`. Some up-to-date diagrams can be see here: https://github.com/LiskHQ/lisk-contracts-private/blob/main/documentation/staking-governance.md#contracts-overview
But for this documentation of the governance system, I would avoid explaining the separation of the different staking contracts. So, it might be fine if you just talk about the rewards contract without any further explanation. Otherwise, you could link to 
- the [implementation](https://github.com/LiskHQ/lisk-contracts-private/blob/main/src/L2/L2Reward.sol),
- the staking docs if there should be some, or
- [this section](https://github.com/LiskHQ/lisk-contracts-private/blob/1ef369eea1d0fe3554a9263bf60c6619f630c0ca/documentation/staking.md#l2reward-contract) from the documentation inside the contracts repository. This PR is, however, not merged yet.

Author: > the staking docs if there should be some

Why not link to [this section](https://github.com/LiskHQ/lisk-contracts-private/blob/main/documentation/staking-governance.md#contracts-overview)? Assuming we will make it public soon.

Reviewer: you could also link to this one if you prefer

Reviewer: I see that you did it already

> Rewards are taken from the balance of the `Staking` constract and calculated on a daily basis, based on the amount locked and the remaining locking duration.

Reviewer: ```suggestion
Rewards are taken from the balance of the `Reward` contract and calculated on a daily basis, based on the amount locked and the remaining locking duration.
```

Author: Was this renamed? It is mentioned as Staking contract [here](https://onchaincollective.atlassian.net/wiki/spaces/Lisk/pages/84574238/Specification+of+the+Lisk+DAO+Contracts)

Reviewer: yes, sorry. See my [answer from above](https://github.com/LiskHQ/lisk-documentation/pull/118#discussion_r1582977129).

Concretely, the weight for a locked amount is given by `lockedAmount * (remainingLockingDurationInDays + 150)`.
The total daily amount of staking rewards, 8,000,000 / 365  LSK, is then shared by all users proportional to their weight.

We plan to use a total of 24 million LSK tokens over the course of 3 years for the LSK staking rewards program (8 million LSK per year).
After the end of this 3 year span, we expect that developments in the Optimism Superchain Ecosystem (shared, decentralized sequencers)  would lead us to a new staking mechanism.

However, even in case the current staking mechanism has to be continued, extra funding could be provided; potential sources are: 

1. the unclaimed LSK tokens (recall they would be moved to Lisk DAO treasury after the 2-years claim window)
2. the staking rewards of the Onchain Foundation, which would be provided to the ecosystem and
3. the profit of the Onchain Foundation from sequencing rewards. 

### Proposals
Proposal are created on [Tally](#tally).
For spam protection, the following requirement must be fulfilled in order to create a proposal:

> - **Proposal threshold**: For creating a proposal, the voting power of the proposer must be at least 300,000 vpLSK, i.e., the value corresponding to 100,000 LSK locked for 2 years where the countdown is paused.

Reviewer: this may also not be completely clear without reading the "voting power" section before. I would propose to move the "Proposals" section to after "Delegation" (but before "Staking rewards" as this one is the least important one on this page in my opinion).

#### Implementation of general proposals
For approved general proposals, the Onchain Foundation will determine whether the proposal is safe, secure, consistent with the scope of Lisk governance, and capable of being implemented in a legally compliant manner.
If it is, the Foundation will act diligently and in a commercially reasonable manner to cause the proposal to be implemented.

### Voting Power
To get some voting power, users must **lock/stake** some LSK tokens. This works as follows:

Users can lock tokens for a specific locking duration (between 2 weeks and 2 years).
After the locking duration ends, the users can redeem their tokens.
Locking happens on our portal.

Initially, the voting power of a locked token will be constant during its locking duration.
But it is possible to [boost the voting power](#boosting-the-voting-power) by up to 200% where the boost is proportional to remaining locking duration.

We consider to transition to a voting-escrow-Token model at some point.
In this model the voting power decreases linearly when getting closer to end of the locking duration.

> But this requires custom integration from Tally.

Reviewer: It's good to have this information on confluence for us internally, but not for the public. I would therefore remove this section.

> #### Voting power calculation:

Reviewer: ```suggestion
#### Voting power calculation
```

Locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides one unit of voting power.

> #### Boosting the voting power

Reviewer: I would make it a subsection of "Voting power calculation"

#### Benefits of being a delegate

- **Impact**: 
  - Have influence on the direction of the Lisk project.
  - Decide directly about fund allocation of the Lisk DAO treasury.
  {/* TODO: Confirm rewards */}
{/* - **Rewards**:
  - Being a delegate is required for some LSK air drops.
  - Being a delegate can be benefitial e.g. when applying for the Grant or Ambassador programs.
  - Receive extra rewards for being an excellent delegate (e.g., being most active/constructive, having most delegations, etc). */}
- **Reputation**:
  - Gain reputation by being a delegate.

### Administrative role of the Onchain Foundation
The foundation will have an administrative role and will take care of the following

1. Removal of draft proposals that reasonably appear to be fraudulent, spam-oriented, defamatory, hateful, or otherwise inappropriate.
2. Management of mutually contradictory proposals that are submitted simultaneously or in close proximity to one another.
3. Administration of network maintenance, such as emergency bug fixes or release rollbacks (with or without a governance vote).

## Governance Goals
There are two primary goals of Lisks governance system:

> ### Capture resistance

Reviewer: I'm not sure I would keep this section as we do not really achieve it.

Author: Currently not, but wouldn't this be a goal we want to achieve, even if it is in far future?

Governance should:

1. make it possible for the chain or network operations to continue without reliance on any individual entity, and 
2. prevent any one entity or small group of entities from being able to control or censor the protocol or its functions.

### Resource allocation
Governance’s second primary responsibility is to allocate resources effectively to support the Lisks vision and accrue sustainable value to the Lisk DAO.
Vision & value may often be in conflict, and allocating resources effectively involves a blend of short- and long-term thinking.
This includes allocation of both the token treasury and protocol revenue.

## Design Principles
Design decisions for Lisks governance system should be made in line with three key principles:

### Governance minimization
The set of governance responsibilities that are encoded onchain or formalized in voting processes should remain as minimal as possible.
The Lisk DAO aims to reduce governance to its essence and to avoid introducing regulation where freedom can achieve the same result.
This principle is key to encouraging permissionless innovation.
In practice, this looks like a minimal set of:

> 1. onchain governance processes to upgrade Lisk contracts and tune the economic parameters of the system, and

Reviewer: > onchain governance processes to upgrade Lisk contracts and tune the economic parameters of the system

This might be misleading. We do have onchain voting for it, but in a binding way. For instance, we could have a proposal to upgrade the staking contract to a new version. But the proposal would not be binding the founndation could ignore it.

Author: True. I tried to formulate it clearer, please check.

Reviewer: sounds good to me

2. offchain social processes to maintain a healthy community.

> ### Iteration

Reviewer: Is it really the idea to reduce the role of the foundation over time? I'm not sure what's the plan, but I also never heard that this is the goal.

Author: I took this from the OP governance design principles. I am also not fully aware if this is a design principle we have for the Lisk DAO, but imho this should be a goal for every DAO to become more decentralized over time.
Who should we ask to clarify this ?

---
title: Governance
sidebar_position: 1
slug: /governance
description: Documentation about the governance of the Lisk project.
keywords:
  [
    Lisk governance,
    Lisk DAO,
    Lisk Grants,
    Funding,
    Delegation,
    Voting,
    Proposals,
  ]
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

# Governance

At the moment, the security of the Lisk Mainnet is dependent on a [multisig wallet](https://www.coindesk.com/tech/2020/11/10/multisignature-wallets-can-keep-your-coins-safer-if-you-use-them-right/) managed by several individuals of the Onchain Foundation.
This multisig wallet can be used to upgrade core Lisk Mainnet smart contracts without upgrade delays.

To decentralize the governance of the Lisk project further, Lisk introduces a decentralized autonomous organization, the Lisk DAO, which allows Lisk token holders to participate together in the governance of the Lisk project.

:::info

> Every LSK token holder who participates in the DAO is rewared with [staking rewards](#staking-rewards), to incentivize participation in the Lisk governance.

Reviewer 2: ```suggestion
Every LSK token holder who participates in the DAO is rewarded with [staking rewards](#staking-rewards), to incentivize participation in the Lisk governance.
```

:::

## Lisk DAO
The Lisk DAO provides the opportunity for LSK token holders to participate in the overall governance of the Lisk project, including allocation of funds, protocol updates, or strategic directions. Decisions are made via [proposals](#proposals).

### Tools
The following tools are available to participate in the Lisk DAO:

<Tabs>
  <TabItem value="tally" label="Tally">
  Tally is the platform used to create and vote for proposals of the Lisk DAO.

  {/* TODO: Update link to tally instance */}

>   The [Lisk DAO Tally instance](https://www.tally.xyz/gov/3rd-testing) is the place where users can vote on proposals or delegate their voting power.

Reviewer 2: What is 3rd-testing?

Author: its a Tally instance Matjaz set up for testing. I added a TODO to update it with the link of the final Tally instance, once we have it

  The `#lisk-dao` channel in the [Lisk Chat](https://lisk.chat) Discord server is a community chat dedicated to discuss topics all around the Lisk DAO, share news and updates, or share ideas for new proposals.
  </TabItem>
</Tabs>

### Treasury
{/* TODO: Update link to timelock contract */}
The Lisk DAO has an associated treasury, which resides in the [Timelock contract](https://sepolia-blockscout.lisk.com/address/0x4b727D3f74DD4c124Ced030335DebD5e7EBB8366).
The current balance of the treasury can be checked on the [Lisk BlockScout Explorer](https://sepolia-blockscout.lisk.com/address/0x4b727D3f74DD4c124Ced030335DebD5e7EBB8366).
The treasury is NOT managed by the Onchain Foundation.
Instead, the funds from the treasury can only be transferred via successful [funding proposals](#proposal-types).

Tokens for the Lisk DAO fund are minted newly and added to the Lisk DAO according to the following vesting plan:

| Year | Amount (LSK) | Description |
|------|--------------|-------------|
| 2024 | 15,000,000 | 6,250,000 LSK liquid at migration, 8,750,000 LSK are linearly released in 2024, starting from the migration. |
| 2025 | 15,000,000 | LSK are linearly released over the year. |

> | 2026 | 15,000,000 | LSK are linearly released over the year. |

Reviewer 2: ```suggestion
| 2025 | 15,000,000 | LSKs are linearly released over the year. |
| 2026 | 15,000,000 | LSKs are linearly released over the year. |
```

Author: It is both LSK singular and plural

:::note

> The DAO fund may be increased further in the future by additional 100,000,000 LSK over the following 7 years, if the DAO members approve the corresponding proposals.

Reviewer 2: ```suggestion
The DAO fund may be increased further in the future by an additional 100,000,000 LSK over the following 7 years if the DAO members approve the corresponding proposals.
```

:::

### Voting power
To get some voting power, users must **stake/lock** some LSK tokens.

This works as follows:

- Users can stake tokens for a specific locking duration (between 2 weeks and 2 years).
- Users receive voting power and [staking rewards](#staking-rewards) proportional to the amount of locked tokens.
- After the locking duration ends, the users can redeem their tokens.

Staking happens on the [Lisk portal](https://portal.lisk.com/).

Initially, the voting power of a locked token will be constant during its locking duration.

> But it is possible to [boost the voting power](#boosting-the-voting-power) by up to 200% where the boost is proportional to remaining locking duration.

Reviewer 2: ```suggestion
But it is possible to [boost the voting power](#boosting-the-voting-power) by up to 200% where the boost is proportional to the remaining locking duration.
```

#### Voting power calculation
Locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides one unit of voting power.

##### Boosting the voting power
To enable voting power proportional to the promised locking duration, users have the option to pause the countdown of the locking period.
That means, the counting towards the end of the locking period is paused until the user decides to resume it.
In the case of pausing, the user receives a boost of the voting power. It is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`.
Hence, the voting power can be increased by up to 200%.

### Delegation

> The Governor framework requires to **delegate the voting power** in order to use it.

Reviewer 2: ```suggestion
The Governor framework requires **delegating the voting power** in order to use it.
```

The voting power can be delegated to other users or to themselves, but self-delegation is a requirement if users want to vote themselves.

> Delegation happens on [Tally](#tally).

Reviewer 2: Hyperlink doesn't work!

Author: fixed it

#### Benefits of being a delegate

- **Impact**: 
  - Have influence on the direction of the Lisk project.
  - Decide directly about fund allocation of the Lisk DAO treasury.
  {/* TODO: Confirm rewards */}
{/* - **Rewards**:
  - Being a delegate is required for some LSK air drops.
  - Being a delegate can be benefitial e.g. when applying for the Grant or Ambassador programs.
  - Receive extra rewards for being an excellent delegate (e.g., being most active/constructive, having most delegations, etc). */}
- **Reputation**:

>   - Gain reputation by being a delegate.

Reviewer 2: ```suggestion
  - Gain a reputation by being a delegate.
```

{/* TODO: Create guide */}
{/* For more information, check the guide on [How to delegate your voting power] */}

### Proposals

> Proposals are created on [Tally](#tally).

Reviewer 2: Hyperlink doesn't work!

#### Proposal types

| Type        | Description   | Binding?      |
| :---------- |:------------- |:------------- |
| **Funding** | A proposal for receiving a certain amount of funds to an address. If approved, the recipient receives the amount given in the proposal. | Yes|
| **General** |  General proposals about protocol parameters or the project direction in general that may be followed by the Onchain Foundation or not, depending also on feasibility and cost.  | No  |

#### Proposal evaluation
The following requirements must be fulfilled for a proposal to pass:

- **Quorum**: For a proposal to pass, the “yes” and “abstain” votes must sum up at least to 24,000,000 vpLSK, i.e., the value that corresponds to 8,000,000 LSK locked for 2 years where the countdown is paused.
- There must be strictly more “yes” than “no” votes.

#### Implementation of funding proposals
The transfer from the [Lisk DAO treasury](#treasury) can be executed by anyone, once a funding proposal is approved.

> Funding proposals are **binding**, which means that no-one, not even the Onchain Foundation, can prevent the transfer, if the proposal passed.

Reviewer 2: ```suggestion
Funding proposals are **binding**, which means that no one, not even the Onchain Foundation, can prevent the transfer if the proposal passes.
```

#### Implementation of general proposals
For approved general proposals, the Onchain Foundation will determine whether the proposal is safe, secure, consistent with the scope of Lisk governance, and capable of being implemented in a legally compliant manner.
If it is, the Foundation will act diligently and in a commercially reasonable manner to cause the proposal to be implemented.

### Administrative role of the Onchain Foundation
The foundation will have an administrative role and will take care of the following

1. Removal of draft proposals that reasonably appear to be fraudulent, spam-oriented, defamatory, hateful, or otherwise inappropriate.
2. Management of mutually contradictory proposals that are submitted simultaneously or in close proximity to one another.
3. Administration of network maintenance, such as emergency bug fixes or release rollbacks (with or without a governance vote).

### Staking rewards

> Staking rewards incentivize users to participate in governance, making the decision making more decentralized and community-driven.  

Reviewer 2: ```suggestion
Staking rewards incentivizes users to participate in governance, making the decision-making more decentralized and community-driven.  
```

Author: rewards are plural

In short, users get [voting power](#voting-power) and rewards for locking/staking tokens for a specific time period.

The Lisk DAO applies a **time-weight boosting** on rewards:
The further the end of the locking duration is in the future, the more rewards you get.
This also implies that the rewards decrease every day.
This way, we reward token holders for committing long-term to the Lisk project so we attract them as long-term advocates for Lisk while reducing short-term speculation and volatility. 

- Users can modify their locking positions (increase amount, extend/pause locking duration) at any time they want.
- Users can claim their rewards at any time.   
- Users have the option to **pause** their locking period countdown.
That means, the remaining locking duration remains fixed until the user decides to resume it.
Note that this results in **higher staking rewards**, since the weight remains fixed, as opposed to a weight daily decreasing when the countdown is active. 

#### Staking rewards distribution
{/* TODO: Add Blockscout link to address of the Staking contract */}

> Rewards are taken from the balance of the `Staking` constract and calculated on a daily basis, based on the amount locked and the remaining locking duration.

Reviewer 2: ```suggestion
Rewards are taken from the balance of the `Staking` contract and calculated daily, based on the amount locked and the remaining locking duration.
```

Concretely, the weight for a locked amount is given by `lockedAmount * (remainingLockingDurationInDays + 150)`.
The total daily amount of staking rewards, 8,000,000 / 365  LSK, is then shared by all users proportional to their weight.

We plan to use a total of 24 million LSK tokens over the course of 3 years for the LSK staking rewards program (8 million LSK per year).

> After the end of this 3 year span, we expect that developments in the Optimism Superchain Ecosystem (shared, decentralized sequencers)  would lead us to a new staking mechanism.

Reviewer 2: ```suggestion
After the end of this 3-year span, we expect that developments in the Optimism Superchain Ecosystem (shared, decentralized sequencers) will lead us to a new staking mechanism.
```

However, even in case the current staking mechanism has to be continued, extra funding could be provided; potential sources are: 

1. the unclaimed LSK tokens (recall they would be moved to Lisk DAO treasury after the 2-years claim window)
2. the staking rewards of the Onchain Foundation, which would be provided to the ecosystem and

> 3. the profit of the Onchain Foundation from sequencing rewards. 

Reviewer 2: ```suggestion
1. The unclaimed LSK tokens (such tokens will be moved to the Lisk DAO treasury after the 2-year claim window).
2. The staking rewards of the Onchain Foundation, which would be provided to the ecosystem.
3. The profit of the Onchain Foundation from sequencing rewards. 
```

Author: ```suggestion
1. Unclaimed LSK tokens 
2. The staking rewards of the Onchain Foundation, which would be provided to the ecosystem and
3. The profit of the Onchain Foundation from sequencing rewards. 
```

## Governance Goals

> There are two primary goals of Lisks governance system:

Reviewer 2: ```suggestion
There are two primary goals of the Lisk's governance system:
```

Author: adding `the` sounds wrong here

### Capture resistance
Governance plays a key role in securing the anti-capture and censorship resistance of the Lisk protocol.
Governance should:

1. make it possible for the chain or network operations to continue without reliance on any individual entity, and 
2. prevent any one entity or small group of entities from being able to control or censor the protocol or its functions.

### Resource allocation
Governance’s second primary responsibility is to allocate resources effectively to support the Lisks vision and accrue sustainable value to the Lisk DAO.
Vision & value may often be in conflict, and allocating resources effectively involves a blend of short- and long-term thinking.

## Design Principles

> Design decisions for Lisks governance system should be made in line with three key principles:

Reviewer 2: ```suggestion
Design decisions for Lisk's governance system should be made in line with three key principles:
```

### Governance minimization
The set of governance responsibilities that are encoded onchain or formalized in voting processes should remain as minimal as possible.
The Lisk DAO aims to reduce governance to its essence and to avoid introducing regulation where freedom can achieve the same result.
This principle is key to encouraging permissionless innovation.
In practice, this looks like a minimal set of:

1. onchain governance processes([binding](#implementation-of-funding-proposals) and non-binding) to upgrade Lisk contracts or allocate funds, and
2. offchain social processes to maintain a healthy community.

### Iteration

> Lisk is decentralizing iteratively to increase the chances of building a healthy system that lasts for the long-term.

Reviewer 2: ```suggestion
Lisk is decentralizing iteratively to increase the chances of building a healthy system that lasts for the long term.
```

> This means the Foundation will play a role in establishing processes, help the DAO through its first few rapid feedback loops in improving those processes, then reduce its role over time.

Reviewer 2: ```suggestion
This means the Foundation will play a role in establishing processes, help the DAO through its first few rapid feedback loops in improving those processes, and then reduce its role over time.
```

(This also means the design principles and goals outlined in this document may be invalidated or updated along the way.)
This iteration gives the DAO a chance to learn how to make thoughtful decisions using an un-intuitive but essential loop: introduce a governance process that involves active participation, then gradually work to automate or minimize it over time.
Governance’s responsibility then becomes to adjust the autopilot when necessary, not to keep two hands on the wheel.

### Forking 
The ability to fork and the ability to exit are critical to protect individual freedoms.
All of the core software and tooling required to run the Lisk network should be made open source, freely available, and easy to use such that a fork is always a viable alternative.

> This isn’t just about vibes: in crypto, where credible commitments not to extract are what makes decentralized platforms valuable, this is a competitive advantage.

Reviewer 2: I don't understand the statement, maybe re-word it?

Author: Agree it sounds a little confusing, I removed the line

> This isn’t just about vibes: in crypto, where credible commitments not to extract are what makes decentralized platforms valuable, this is a competitive advantage.

Author: ```suggestion
```

| 2026 | 15,000,000 | LSK are linearly released over the year. |

:::note
The DAO fund may be increased further in the future by an additional 100,000,000 LSK over the following 7 years if the DAO members approve the corresponding proposals.
:::

### Voting power
To get some voting power, users must **stake/lock** some LSK tokens.

This works as follows:

- Users can stake tokens for a specific locking duration (between 2 weeks and 2 years).
- Users receive voting power and [staking rewards](#staking-rewards) proportional to the amount of locked tokens.
- After the locking duration ends, the users can redeem their tokens.

Staking happens on the [Lisk portal](https://portal.lisk.com/).

Initially, the voting power of a locked token will be constant during its locking duration.
But it is possible to [boost the voting power](#boosting-the-voting-power) by up to 200% where the boost is proportional to the remaining locking duration.

#### Voting power calculation
Locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides one unit of voting power.

##### Boosting the voting power

> To enable voting power proportional to the promised locking duration, users have the option to pause the countdown of the locking period.

Reviewer: > To enable voting power proportional to the promised locking duration

I would remove this part as the current version does not give any hint anymore that we ever aimed for linearly decreasing voting power.

:::

### Voting power

> Every account holding **LSK** is able **stake/lock** a specific amount of LSK tokens to get some voting power and [staking rewards](#staking-rewards).

Reviewer: ```suggestion
Every account holding **LSK** is able to **stake/lock** a specific amount of LSK tokens to get some voting power and [staking rewards](#staking-rewards).
```

### Delegation
The Governor framework requires **delegating the voting power** in order to use it.
The voting power can be delegated to other users or to yourself.
Self-delegation is a requirement if you want to vote on your own.

Delegation happens on [Tally](#tools).

#### Importance of delegation

> Delegation is a key feature of the Lisk DAO, as it allows token holders to participate in the governance of the Lisk project passively by entrusting a representative to vote on proposals in a way that aligns with their own beliefs and values.

Reviewer: I think it must be "entrusting a representative with voting on ..."

> By delegating their voting power to a delegate of their choice, token holders can ensure that their interests are represented in the decision-making process.

Reviewer: > token holders can ensure that their interests are represented

This must not necessarily be true as the delegate may vote against the interest of the delegater. You could change the wording "ensure" to something softer (I don't have a good proposal that is short) or just remove the sentence.

Author: replaced it with "influence" now

However, token holders give a lot of power to the delegate to make the right decisions on proposals.

> This is why it is important to choose a delegate that you trust and that you believe will act in the best interest of the Lisk project and the community.

Reviewer: > This is why

Due to this formulation, I would not start a new paragraph here. Up to you.

We recommend to choose a delegate that is active in the community, has a good reputation, and has a track record of making good decisions.

The following sources might support you in your delegation decision:

- [Lisk Chat](https://lisk.chat): Get in touch with your delegate in the Lisk chat, or ask other users about their experiences with the delegate.
- [Lisk Governance Forum](https://forum.lisk.com/c/delegates/): The Lisk Governance Forum provides a section for delegates to present themselves to the community.
- **Lisk DAO Tally instance**: Check the delegate's profile, voting history and activity on the Lisk DAO Tally instance.
{/* TODO: Uncomment link on DAO launch */}
{/* - [Lisk DAO Tally instance](https://www.tally.xyz/gov/3rd-testing): */}

#### How to delegate your voting power

The Lisk DAO uses Tally to facilitate delegation. 

The process in a nutshell:

1. Connect your wallet to Tally and navigate to the Lisk DAO page.

> 2. Select the "delegate" button.

Reviewer: What's missing is that one must first go to the  "my voting power" page to see the "delegate" button. And then after selecting "Someone else", one must click on the slightly hidden button "Explore all Delegates" to see the overview. Therefore, we may just provide the link to the "delegates" page directly. Which is `https://www.tally.xyz/gov/<DAO name>/delegates`

3. Search for the delegate you'd like to delegate your voting power to and confirm the delegation by following Tally's prompts.

:::tip
Through Tally, you can change or revoke your delegation at any time. 

> You can also proportionally delegate your voting power to multiple delegates. 

Reviewer: No. This is not possible.

Author: oh you are right. It was described like this in the Arbitrum docs. Maybe Tally had this feature in the past?

Reviewer: I don't think that Tally ever had this feature, because the underlying contracts do not allow it.

:::

#### Responsibility of delegates
Delegates play a crucial role in the governance of the Lisk project.
They are the ones who create proposals, vote on behalf of the token holders, and represent the interests of the community as a whole.

Delegates are required to:

- Follow the processes outlined in the Governance documentation of the Lisk DAO
- Act in the best interest of the token holders whom they represent
- Be active in the decision-making process of the DAO, this means specifically:
  - Vote on proposals

>   - Create proposals

Reviewer: I would remove "create proposal" as this sounds like it expected from every delegate which is not the case.

>   - Give feedback on other draft proposals

Reviewer: Also for this one I'm not sure we should formulate it as a requirement. Average delegates may be able to make well-informed voting decision for most proposal. But providing feedback on draft proposals is much harder and may only make sense in some cases.

Author: i added a new paragraph below

{/* TODO: Create guide */}
{/* For more information, check the guide on [How to delegate your voting power] */}

#### How to become a delegate

> Basically anyone who owns some LSK tokens can become a delegate.

Reviewer: I find it a bit misleading, because one can also become a delegate without having any LSK or voting power.

Author: rephrased it, please check

Reviewer: I would shorten lines 89-95 to the following:

> Basically anyone can become a delegate. There are various ways to do this:

Reasons:
- self-delegation is not a requirement for becoming a delegate.
- reaching the proposal threshold is not the only (and probably also not the main) motivation to become a delegate

Author: I would disagree here.
First, is it a realistic/common scenario for people who have no own LSK to be delegate? I mean, I don't see how it is realistic for them to receive delegations from others. Why would I trust someone who doesn't have own stakes in the project? Additionally, I don't see any good motivation for someone not holding LSK to be in the DAO.
So yes, in fact I think the first logical step for every delegate is to stake and self-selegate their voting power. And then, depending ont heir motivations, they can decide to advocate for delegations from other users.

Reviewer: It's probably not very common, but it exists. [L2beat](https://www.tally.xyz/gov/arbitrum/delegate/l2beatcom.eth), for instance, has no ARB tokens but is one of the largest delegates in Arbitrum.
So, for some entities, it might be obvious that they vote for the best interest of the project without staking and therefore users may choose them as a delegate. This could, for example, also be some projects deploying on Lisk.

And even if L2Beat on Arbitrum is the only exception, I would not write something that is technically misleading.

Author: I adjusted the section, please check :)

### Resource allocation
Governance’s second primary responsibility is to allocate resources effectively to support the Lisks vision and accrue sustainable value to the Lisk DAO.
Vision & value may often be in conflict, and allocating resources effectively involves a blend of short- and long-term thinking.

## Design Principles
Design decisions for Lisk's governance system should be made in line with three key principles:

### Governance minimization
The set of governance responsibilities that are encoded onchain or formalized in voting processes should remain as minimal as possible.
The Lisk DAO aims to reduce governance to its essence and to avoid introducing regulation where freedom can achieve the same result.
This principle is key to encouraging permissionless innovation.
In practice, this looks like a minimal set of:

> 1. onchain governance processes([binding](#implementation-of-funding-proposals) and non-binding) to upgrade Lisk contracts or allocate funds, and

Reviewer: ```suggestion
1. onchain governance processes ([binding](#implementation-of-funding-proposals) and non-binding) to upgrade Lisk contracts or allocate funds, and
```
